lib/lib-googlecloud/src/main/java/org/sagebionetworks/googlecloud/SynapseGoogleCloudStorageClient.java

```diff
@@ -67,4 +83,27 @@
 	 * @param newKey the new name of the object
 	 */
 	void rename(String bucket, String oldKey, String newKey);
+
+	/**
+	 * Gets the list of Blobs in a bucket with a particular prefix.
+	 * @param bucket the bucket containing objects to return
+	 * @param keyPrefix the prefix that objects should match
+	 * @return
+	 */
+	List<Blob> getObjects(String bucket, String keyPrefix);
```

Comment:
needs to be a paginated iterator.

Reply:
Google's `Page<T>` (the object they return in their API) provides a method `iterateAll()` that returns `Iterable<T>`, which lazy-loads the next page when it is reached. Will use that object.